packages/edit-site/src/components/sidebar-navigation-screen-patterns/index.js

```diff
@@ -72,7 +75,7 @@ function ThemePatternsGroup( { categories, currentCategory, currentType } ) {
 								<Tooltip
 									position="top center"
 									text={ __(
-										'Theme patterns cannot be edited.'
+										'Patterns bundled with themes or plugins cannot be edited.'
```

Comment:
We should just add the plugin reference like this: 

"Theme and plugin patterns cannot be edited."

Reply:
That, or "Third-party patterns cannot be edited"

Reply:
cc @annezazu thoughts?

Reply:
@annezazu thoughts?

Reply:
Agreed. That feels much cleaner and simpler.